src/SFA.DAS.Authorization.UnitTests/ProviderPermissions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.Authorization.UnitTests
+{
+    public struct ProviderPermissions
```

Comment:
Instead of recreating this in the unit test project, we should probably use the source class (SFA.DAS.Authorization.ProviderPermissions/ProviderPermissions.cs).